src/NotificationListener.php

```diff
@@ -99,6 +100,17 @@ public function listen(MpxMediaSourceInterface $media_source, int $notification_
         return [];
       }
 
+      if ($e->getCode() == 404) {
+        $this->logger->warning(
+          'The last notification ID %id for %account is older than 7 days and is too old to fetch notifications. The last notification ID has been reset to re-start ingestion of all videos.',
+          [
+            '%id' => $notification_id,
+            '%account' => $account->getPid(),
```

Comment:
Since we have the media source, can we make this easier to trace? The public id is a random set of letters. Instead, it could be `$media_source->getAccount()->label()` which would be the label of the account config entity.